build/angular-12/index.ts

```diff
@@ -1,14 +0,0 @@
-import { ngPackagr } from 'ng-packagr';
-import { buildPackages, getOptionsFromProcessArgs } from '../../tools/build-packages';
-import { setMetadata } from '../../tools/set-metadata';
-
-const options = getOptionsFromProcessArgs();
-
-buildPackages(options, ngPackagr)
```

Comment:
The purpose of this folder and file was so that we can build with a specific version of angular to ensure the best compatibility, and to have a `node_modules` folder here that is isolated from the root `node_modules` folder, so that this build process is as untainted as possible.
Have we lost this ability with the move to nx, or does it now take a different form?
PS. I am not specifically requesting this ability to be preserved, but I am just trying to ascertain what abilities are built in or not.

Reply:
From my understanding packages were always built with 12th version of angular from main node_modules, isn't it correct? Right now each package has it's own build target that runs `@angular-devkit/build-angular:browser` executor of version 12. I intentionally didn't migrate those to `@nrwl/angular` targets so that we're not tied to the version of nx that is used in the repo.

Reply:
That is really awesome. Thank you!